ple/games/vgdlple.py

```diff
@@ -0,0 +1,168 @@
+import sys
+import pygame
+import base
+import vgdl.core as core
+
+# Hacked together VLE wrapper for py-VGDL. Potentially extremely buggy - use with caution.
+# This allows you to use VGDL (see http://people.idsia.ch/~tom/publications/pyvgdl.pdf , thanks to Tom Schaul) with the PLE.
+
+# Example VGDL description text
+# The game dynamics are specified as a paragraph of text
+aliens_game = """
+BasicGame
+    SpriteSet
+        base    > Immovable    color=WHITE
+        avatar  > FlakAvatar   stype=sam
+        missile > Missile
+            sam  > orientation=UP    color=BLUE singleton=True
+            bomb > orientation=DOWN  color=RED  speed=0.5
+        alien   > Bomber       stype=bomb   prob=0.01  cooldown=3 speed=0.75
+        portal  > SpawnPoint   stype=alien  cooldown=16   total=20
+    
+    LevelMapping
+        0 > base
+        1 > portal
+
+    TerminationSet
+        SpriteCounter      stype=avatar               limit=0 win=False
+        MultiSpriteCounter stype1=portal stype2=alien limit=0 win=True
+        
+    InteractionSet
+        avatar  EOS  > stepBack
+        alien   EOS  > turnAround        
+        missile EOS  > killSprite
+        missile base > killSprite
+        base missile > killSprite
+        base   alien > killSprite
+        avatar alien > killSprite
+        avatar bomb  > killSprite
+        alien  sam   > killSprite         
+"""
+
+# the (initial) level as a block of characters 
+aliens_level = """
+wwwwwwwwwwwwwwwwwwwwwwwwwwwwwwww
+w                              w
+w1                             w
+w000                           w
+w000                           w
+w                              w
+w                              w
+w                              w
+w                              w
+w    000      000000     000   w
+w   00000    00000000   00000  w
+w   0   0    00    00   00000  w
+w                A             w
+wwwwwwwwwwwwwwwwwwwwwwwwwwwwwwww
+"""
+
+
+class vgdlPLE(base.PyGameWrapper):
+
+
+    def __init__(self, game_str = aliens_game, map_str = aliens_level):
+
+	# GVG-AI competition allows 40ms per move, which equates to 25fps
+        self.allowed_fps = 25
```

Comment:
Physics update stepwise rather than by a difference from the last update. I'll need to check how this works with the ContinuousPhysics games, but I'm fairly sure it's the same.